vmss-prototype/vmss-prototype

```diff
@@ -830,6 +829,25 @@ def vmss_prototype_update(sub_args):
                 '--name', extension['name']
                 ], retries=3, check=False, retry_func=not_found_no_retry)
 
+    # Scale out 1 more VMSS instance to replace the previously deleted instance
+    # Note: this does not respect any changes that may have occured to the VMSS instance count during update
+    # from out-of-band node scaling tools such as cluster-autoscaler
+    output, _, _ = run(az(['vmss', 'show'], subscription) + [
+                       '--resource-group', resource_group,
+                       '--name', vmss
+                       ], retries=3)
+    vmss_info = json.loads(output)
+    capacity = vmss_info["sku"]["capacity"]
```

Comment:
If you are worried that capacity may not get returned, we should use

`capacity = lookup_one_value(vmss_info,'sku.capacity')`

That will return None if the value does not exist.